csharp/ql/src/semmle/code/csharp/serialization/Deserializers.qll

```diff
@@ -52,6 +53,11 @@ class MicrosoftDeserializer extends UnsafeDeserializer {
   }
 }
 
+/** An unsafe deserializer method in the `Newton.Json.*` namespace. */
```

Comment:
The class name and the comment should mention `Newtonsoft.Json` instead of `Newton.Json`.